metpy/calc/thermo.py

```diff
@@ -308,8 +308,10 @@ def _lcl_iter(p, p0, w, t):
 
 
 @exporter.export
-@check_units('[pressure]', '[temperature]', '[temperature]')
-def lfc(pressure, temperature, dewpt):
+@check_units('[pressure]', '[temperature]', '[temperature]', '[temperature]',
+             '[pressure]')
+def lfc(pressure, temperature, dewpt, parcel_temperature_profile=None,
+        parcel_starting_pressure=None):
```

Comment:
Yeah, let's just remove this and rely on `pressure` and `parcel_temperature_profile` being in agreement. If/when we have a concrete use case for something more complicated, we can investigate at that time.